test/podFailover/controller/cmd/main.go

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sync"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/retry"
+	"k8s.io/klog/v2"
+	podfailure "sigs.k8s.io/azuredisk-csi-driver/test/podFailover/api"
+)
+
+const (
+	podFailoverLabelKey    = "app"
+	podFailoverLabelValue  = "pod-failover"
+	sameNodeFailoverConst  = "same-node-failover"
+	deletePodFailoverConst = "delete-pod"
+)
+
+func main() {
+	config, err := rest.InClusterConfig()
```

Comment:
I see you added support for the `--kubeconfig` parameter to the workload binary. We should do the same here in a subsequent PR.